cypress/e2e/yves/shop/locale.cy.ts

```diff
@@ -0,0 +1,64 @@
+import { container } from '@utils';
+import { LocaleScenario } from '@scenarios/yves';
+import { CatalogPage, IndexPage } from '@pages/yves';
+import { LocaleStaticFixtures } from '@interfaces/yves';
+
+(['b2c', 'b2c-mp', 'b2b', 'b2b-mp'].includes(Cypress.env('repositoryId')) ? describe.skip : describe)(
+  'locale switching',
+  { tags: ['@shop', '@locale'] },
```

Comment:
```suggestion
  { tags: ['@yves', '@core'] },
```
Rename also folder from shop to core (in order to be consistent with spryker features).